components/PodcastContainer.jsx

```diff
@@ -10,6 +10,8 @@ class PodcastContainer extends React.Component {
       title: PropTypes.string,
       link: PropTypes.string,
     })).isRequired,
+    listHeight: PropTypes.number.isRequired,
```

Comment:
I think that having listHeight be defined by the Podcast container instead of passed through the entire hierarchy may be more advantageous here. PodcastList could still take advantage of dynamic height when wrapped by a different container